grafana/dashboards/cmode/workload.json

```diff
@@ -3502,7 +3623,7 @@
             },
             {
               "exemplar": false,
-              "expr": "label_join(\n  qos_workload_max_throughput_mbps{cluster=~\"$Cluster\",datacenter=~\"$Datacenter\",is_adaptive=\"Yes\",workload=~\"$Workload\"},\n  \"unique_id\",\n  \"-\",\n  \"datacenter\",\n  \"cluster\",\n  \"workload\"\n)",
+              "expr": "label_join(\n    (qos_total_data{cluster=~\"$Cluster\",datacenter=~\"$Datacenter\",workload=~\"$Workload\"} * 0)\n  + on (cluster, datacenter, workload) group_right ()\n    qos_workload_max_throughput_mbps{cluster=~\"$Cluster\",datacenter=~\"$Datacenter\",is_adaptive=\"Yes\",workload=~\"$Workload\"},\n  \"unique_id\",\n  \"-\",\n  \"datacenter\",\n  \"cluster\",\n  \"workload\"\n)",
```

Comment:
See if below is simpler and can be applied at other places for similiar pattern.

```
label_join(
  qos_workload_max_throughput_iops{
    cluster=~"$Cluster",
    datacenter=~"$Datacenter",
    is_adaptive="Yes",
    workload=~"$Workload"
  }
  and on(cluster, datacenter, workload)
  qos_ops{
    cluster=~"$Cluster",
    datacenter=~"$Datacenter",
    workload=~"$Workload"
  },
  "unique_id",
  "-",
  "datacenter",
  "cluster",
  "workload"
)
```